src/ttsim/plot/dag/interface.py

```diff
@@ -135,6 +135,8 @@ def interface(
                 else interface_object
             )
             description = inspect.getdoc(f)
+        else:
+            description = ""
```

Comment:
```suggestion
            description = "No description available."
```

And remove the default below.

Reply:
Thx!